docker/server/start.sh

```diff
@@ -27,13 +27,7 @@ echo "================================"
 echo ""
 
 echo ""
-echo "===== Create Database and Import schema ====="
-echo ""
-
-mysql -u $DB_USER -p$DB_PASSWORD -h $DB_HOST -e "CREATE DATABASE $DB_DATABASE;"
-mysql -u $DB_USER -p$DB_PASSWORD -h $DB_HOST -e "SHOW DATABASES;"
-mysql -u $DB_USER -p$DB_PASSWORD -h $DB_HOST -D $DB_DATABASE < schema.sql
```

Comment:
It's better to keep this here, remove the folder 'mysql', and put the mysql image(mariadb:10) direct on the docker-compose file